src/utils/private.ts

```diff
@@ -155,6 +145,10 @@ export const getScalarKey = async (X25519_PRIVATE_KEY: string | undefined): Prom
   }
 };
 
+const mergeConfigs = (configs: MergedConfigs) => {
+  return _.merge({}, ...Object.values(configs));
```

Comment:
```suggestion
  return merge({}, configs.parsedDefault, configs.parsedOrg, configs.parsedRepo);
```
It's better to be explicit about ordering because in this case it matters. We don't know how Object.values() will order them.